Classes/Repository/RepositoryIssuesViewController.swift

```diff
@@ -51,10 +58,13 @@ SearchBarSectionControllerDelegate {
         super.viewDidLoad()
 
         makeBackBarItemEmpty()
-
-        // set the frame in -viewDidLoad is required when working with TabMan
-        feed.collectionView.frame = view.bounds
-        feed.collectionView.contentInsetAdjustmentBehavior = .never
+        
+        let presentingInTabMan = label == nil
+        if presentingInTabMan {
+            // set the frame in -viewDidLoad is required when working with TabMan
+            feed.collectionView.frame = view.bounds
+            feed.collectionView.contentInsetAdjustmentBehavior = .never
+        }
```

Comment:
Why do we need this?

Reply:
The top of the view controller was hiding under the nav bar and that fixed it